consensus/src/pipeline/virtual_processor/utxo_validation.rs

```diff
@@ -312,6 +315,8 @@ impl VirtualStateProcessor {
             &mutable_tx.as_verifiable(),
             pov_daa_score,
             TxValidationFlags::SkipMassCheck, // we can skip the mass check since we just set it
+            mutable_tx.calculated_compute_mass,
```

Comment:
Actually this should simply be `contextual_mass` and as such -- not an `Option` but always a `u64`

Reply:
It can be an `Option<u64, f64>`. I think that would be most reflecting of the actual usage

Reply:
Fixed, together with using overall contextual instead of compute mass